uber/site_sections/summary.py

```diff
@@ -142,12 +142,13 @@ def staffing_overview(self, session):
             } for dept, desc in c.JOB_LOCATION_OPTS]
         }
 
+    # print out a CSV list of department heads with their associated departments and contact information
     @csv_file
-    def dept_head_emails(self, out, session):
-        out.writerow(['Name', 'Phone', 'Email'])
-        dhs = session.query(Attendee).filter(Attendee.ribbon == c.DEPT_HEAD_RIBBON).all()
-        for x in dhs:
-            out.writerow([x.full_name, x.cellphone, x.email])
+    def department_head_sheet(self, out, session):
+        out.writerow(["Full Name", "Email", "Phone", "Department"])
+        for a in session.query(Attendee).filter_by(staffing=True, placeholder=False, ribbon=c.DEPT_HEAD_RIBBON).order_by('last_name').all():
```

Comment:
I copied that line with staffing=True and placeholder=False from another place. I should go back and find that to update as well. Same with the .all(). I'll happily update. Thanks for the information!